services/base-service/models/expert/model.js

```diff
@@ -47,6 +47,38 @@ class ExpertModel extends BaseModel {
     return s;
   }
 
+  /**
+   * @method sanitize
```

Comment:
@rakunkel-ucd Here is the sanitize step moved into the expert model.  It throws errors, but not `new Error` because it's not a code error, so I don't want a stack trace in the API output